pyzo/core/kernelbroker.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, info=None):
         self.argv = ""
 
         # Additional environment variables
-        self.environ = ""
+        self.environ = "PYZO_PROCESS_EVENTS_WHILE_DEBUGGING=1"
```

Comment:
I don't think this is needed - previously, when the text was the same as the example, it was interpreted as an empty string. 
```suggestion
        self.environ = ""
```